markdown.sile-dev-1.rockspec

```diff
@@ -15,6 +15,7 @@ description = {
 }
 dependencies = {
    "lua >= 5.1",
+   "penlight >= 1.13.0", -- use or kpairs requires newer Penlight than transitive dependency possibly provided by SILE
```

Comment:
Oh ouch! I did not realize that's where the `kpairs` call was, so yes definitely we should note this is SILE itself. I'll take care of that.

Reply:
Actually it already *is* token care of is SILE itself since we're hard pinning dependencies, but the Ubuntu package doesn't take this into account. That will be fixed as soon as my PPA builds go through.